README.md

```diff
@@ -1,24 +1,179 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <img src="./app/assets/images/dilsher-logo.png" alt="logo" width="140"  height="auto" />
+  <img src="./app/assets/images/DSB.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h1><b>Ruby on Rails Budget App</b></h1>
+</div>
+<!-- TABLE OF CONTENTS -->
+ </div>
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the project](#-about-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+    - [🔭 Video Presentation](#video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author : ](#-author--)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+  <div>
+  <!-- PROJECT DESCRIPTION -->
+  # 📖 Blog_App_Rails<a name="about-project"></a>
 
-* Ruby version
+The Ruby on Rails capstone project(ROR-Budget-maker) is a mobile web application where you can manage your budget. You have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* System dependencies
+##🛠 Built With <a name="built-with"></a>
 
-* Configuration
+- Ruby
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li>RSPEC</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
 
-* How to run the test suite
+<!-- Features -->
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- Use database migration files to maintain database schema.
+- Understand ORM and database migrations and their implementation.
+- Processing data in models.
+- Views.
+- Forms
+- Validations and Model specs.
+- Integration specs for Views.
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 🚀 Live Demo <a name="live-demo"></a>
+
+Please click <a href="https://ror-budget-maker.onrender.com/" target="_blank">here</a> to go live with the application.
```

Comment:
- [ ] Great effort with the deployment. Please like the previous reviewer noted, and As required for this project, please ensure that proper functionality of the home and detail pages, creating categories and transactions should work well in the deployed version, unfortunately it's not. Kindly take your time and redeploy, ensure and test if the deployed version is working as expected.
The login, signup and logout are working well but i am unable to create a transaction or category. Please, kindly fix that.
I suggest you to follow this for [netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) or this for [render](https://medium.com/@laura_brooks/how-to-deploy-a-ruby-on-rails-app-with-render-com-9a824193e636)

![Screenshot_27](https://github.com/DilsherB/ror-budget-maker/assets/65298048/91ed7c74-1227-4d29-9aa8-3446246771b4)